src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -449,10 +449,15 @@ void FixedwingAttitudeControl::Run()
 				*/
 				float groundspeed = sqrtf(_local_pos.vx * _local_pos.vx + _local_pos.vy * _local_pos.vy);
 				float gspd_scaling_trim = (_param_fw_airspd_min.get() * 0.6f);
-				float groundspeed_scaler = gspd_scaling_trim / ((groundspeed < gspd_scaling_trim) ? gspd_scaling_trim : groundspeed);
 
 				control_input.groundspeed = groundspeed;
-				control_input.groundspeed_scaler = groundspeed_scaler;
+
+				if (groundspeed > gspd_scaling_trim) {
+					control_input.groundspeed_scaler = gspd_scaling_trim / groundspeed;
+
+				} else {
+					control_input.groundspeed_scaler = 1.0f;
```

Comment:
Actually, if you check how this value is used, it isn't meant to be between 0 and 1 only: https://github.com/PX4/PX4-Autopilot/blob/5bbc66f3afe99e75852e9b3660a5de4d2fc0b902/src/modules/fw_att_control/ecl_wheel_controller.cpp#L86
So if `groundspeed < gspd_scaling_trim` then `groundspeed_scaler` should be >1. We just want to avoid the divide-by-zero; your modification changes the logic.

Reply:
But it's not checking for `<`, but for `groundspeed > gspd_scaling_trim` right now or am I missing something? I usually squash the commits before pushing them back, so maybe this caused some confusion.

Reply:
I was probably tired :D